tbump/config.py

```diff
@@ -126,7 +127,19 @@ def validate_config(cfg: Config) -> None:
 
 
 def parse(cfg_path: Path) -> Config:
-    parsed = tomlkit.loads(cfg_path.text())
+    # tbump.toml used to contain "flat" keys while pyproject.toml contains nested keys.
+    # For example:
+    #
+    #       [[file]]
+    # vs
+    #       [[tool.tbump.file]]
+    #
+    # In order to minimize the number of changes that are needed for this switch
+    # we parse `pyproject.toml` and use it to create a new `tomlkit.Document()` with
+    # a flat structure
+    parsed_pyproject = tomlkit.loads(cfg_path.text())["tool"]["tbump"]
```

Comment:
That's pretty clever but it will crash with KeyError if `[tool]` or `[[tool.tbump]]` section is messing

Reply:
True, but let's not forget that the PR was about implementing Option 2, i.e. only support `pyproject.toml`. In this case, `tool.tbump` should always be present :)))

That being said, handling the `KeyError` would probably make sense anyway.